corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -188,14 +188,23 @@
 <div class="row">
     <div class="col-sm-12">
         <div class="btn-toolbar">
-            <a class="btn btn-primary"
+            <a class="btn btn-info"
+               {% if not has_location_types or not can_edit_any_location %}
```

Comment:
Given that the existing upload/download link is limited to users with edit permissions, this PR isn't making things any worse.

@esoergel should it be possible for someone who can't edit locations to still be able to download them?